internal/runners/reset/reset.go

```diff
@@ -126,16 +127,26 @@ func (r *Reset) Run(params *Params) error {
 		}
 	}
 
-	err = localcommit.Set(r.project.Dir(), commitID.String())
+	err = checkoutinfo.SetCommitID(r.project.Dir(), commitID.String())
 	if err != nil {
 		return errs.Wrap(err, "Unable to set local commit")
 	}
 
 	// Ensure the buildscript exists. Normally we should never do this, but reset is used for resetting from a corrupted
 	// state, so it is appropriate.
 	if r.cfg.GetBool(constants.OptinBuildscriptsConfig) {
-		if err := buildscript_runbit.Initialize(r.project.Dir(), r.auth); err != nil {
-			return errs.Wrap(err, "Unable to initialize buildscript")
+		err := buildscript_runbit.Initialize(r.project.Dir(), r.project.Owner(), r.project.Name(), r.auth)
```

Comment:
You should also have the branch. The project URL we show in the buildscript should be identical to the one in the activestate.yaml.

Reply:
Good call, but that doubled the changeset size :/